### PR TITLE
Remove max width from tag in stepcard, fix elements in card story

### PR DESCRIPTION
**Pull request by haveanicedavid**
## Description

Closes: regen-network/regen-registry#883

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/cards/OverviewCard.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ interface OverviewCardProps {
 
 interface OverviewItem {
   title: string;
-  description: JSX.Element;
+  description: JSX.Element | string;
```
Comment (haveanicedavid) [inline web-components/src/components/cards/OverviewCard.tsx:23]: minor tweak - it renders a blank slot and our stories pass a string

File: web-components/src/components/cards/StepCard.tsx [hunk 2]
```diff
@@ -94,7 +94,6 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginRight: 0,
     borderRadius: 0,
     fontSize: theme.spacing(3),
-    maxWidth: theme.spacing(22),
```
Comment (haveanicedavid) [inline web-components/src/components/cards/StepCard.tsx:97]: only change relevant to the story itself

File: web-components/src/components/cards/card.stories.tsx [hunk 3]
```diff
@@ -31,12 +32,8 @@ export default {
 const projectDeveloper: User = {
   name: 'Odonata',
   type: 'user',
-  place: {
-    city: 'South Melbourne',
-    state: 'Victoria',
-    country: 'Australia',
-  },
-  imgSrc:
```
Comment (haveanicedavid) [inline web-components/src/components/cards/card.stories.tsx:39]: cleanup type errors in the story

**Review by mhagel — APPROVED** [review]

**Comment by haveanicedavid** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**